src/index.js

```diff
@@ -7,9 +7,9 @@ const creator = opts => {
 
 	return {
 		postcssPlugin: 'postcss-place',
-		Declaration: (decl) => {
+		Declaration: (decl, { result }) => {
 			if (placeMatch.test(decl)) {
-				onCSSDeclaration(decl)
+				onCSSDeclaration(decl, { result: result })
```

Comment:
passing `{ result: result }` so that the signature of `onCSSDeclaration` matches `Declaration: (decl, { result }) => {`

This should make it easier if we need to pass more bits from the same object in the future.